games-strategy/freeciv/freeciv-2.5.10.recipe

```diff
@@ -0,0 +1,119 @@
+SUMMARY="A free Civilization-like game"
+DESCRIPTION="Freeciv is a free Civilization-like game, primarily \
+for X under Unix. It has support for multiplayer games locally or \
+over a network, and an AI which gives most people a run for their money.
+Freeciv aims to be mostly rule-compatible with Civilization II [tm], \
+published by Sid Meier and Microprose [tm]. A few rules are different \
+where we think it makes more sense, and we have lots and lots of \
+adjustable parameters to make customizing games possible.
+Freeciv has been implemented completely independently of Civilization; \
+you do not need to own Civilization to play Freeciv."
+HOMEPAGE="http://www.freeciv.org"
+COPYRIGHT="1996-2017 The Freeciv Team"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://downloads.sourceforge.net/project/freeciv/Freeciv%202.5/$portVersion/freeciv-$portVersion.tar.bz2"
+CHECKSUM_SHA256="c8a14156562dce2f0a7b96390c2d5a35314f1124c448782a6a47b0c51cfcb302"
+ADDITIONAL_FILES="freeciv.rdef.in"
```

Comment:
```diff
-ADDITIONAL_FILES="freeciv.rdef.in"
 PATCHES="freeciv-$portVersion.patchset"
+ADDITIONAL_FILES="freeciv.rdef.in"
```
Source: [HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)

BTW, I think `GLOBAL_WRITABLE_FILES` are more frequently added _after_ `BUILD_PREREQUIRES` rather than _before_ `PROVIDES`. (Maybe I should do some metrics and add the mostly used position to the wiki :)